README.md

```diff
@@ -153,6 +153,13 @@ it('Has no a11y violations after button click', () => {
   cy.get('button').click()
   cy.checkA11y()
 })
+
+it('Only logs a11y violations while allowing the test to pass', () => {
+  // Disables assertions based on violations
```

Comment:
Maybe this would be more clear:

```suggestion
  // Do not fail the test when there are accessibility failures
```

Reply:
Yeah, I wasn't sure how to describe that. I like your suggestion